### PR TITLE
CODENVY-959: Get rid of WSocketEventBusClient 

### PR DESCRIPTION
**Pull request by tolusha**
### What does this PR do?
Get rid of WSocketEventBusClient 

### What issues does this PR fix or reference?
https://github.com/codenvy/codenvy/issues/959

### Previous behavior
Ws agent produced enormous number of logs

### New behavior
Get rid of WSocketEventBusClient. It isn't used anymore

Please review [Che's Contributing Guide](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md) for best practices.

### CODE REVIEW

File: core/che-core-api-core/src/main/java/org/eclipse/che/api/core/notification/WSocketEventBusClient.java [hunk 1]
```diff
@@ -254,11 +264,26 @@ public void run() {
...
                     connect(wsUri, channels);
+                    reconnectAttempts = 0;
                     LOG.debug("Connection complete");
                     return;
                 } catch (IOException | DeploymentException e) {
                     LOG.warn("Not able to connect to {} because {}. Retrying ", wsUri, e.getLocalizedMessage());
                     LOG.debug(e.getLocalizedMessage(), e);
+
+                    if (reconnectAttempts++ > MAX_ATTEMPTS) {
+                        LOG.error(WORKSPACE_AGENT_STOPPED_MESSAGE);
```
Comment (skabashnyuk) [inline core/che-core-api-core/src/main/java/org/eclipse/che/api/core/notification/WSocketEventBusClient.java]: log.warn

File: core/che-core-api-core/src/main/java/org/eclipse/che/api/core/notification/WSocketEventBusClient.java [hunk 2]
```diff
@@ -254,11 +264,26 @@ public void run() {
...
+
+                        try {
+                            // content of /etc/hosts file may provide clues on why the connection failed, e.g. how che-host is resolved
+                            LOG.info("Workspace /etc/hosts: " + IoUtil.readAndCloseQuietly(new FileInputStream(new File("/etc/hosts"))));
```
Comment (skabashnyuk) [inline core/che-core-api-core/src/main/java/org/eclipse/che/api/core/notification/WSocketEventBusClient.java]: log.debug

Comment (skabashnyuk) [reply core/che-core-api-core/src/main/java/org/eclipse/che/api/core/notification/WSocketEventBusClient.java]: log.info is too open

File: wsagent/wsagent-local/src/main/java/org/eclipse/che/ApiEndpointAccessibilityChecker.java [hunk 3]
```diff
@@ -63,11 +64,7 @@ public void start() {
             }
 
             LOG.error("The workspace agent has attempted to start, but it is unable to ping the Che server at " + apiEndpoint);
-            LOG.error("The workspace agent has been forcefully stopped. " +
-                      "This error happens when the agent cannot resolve the location of the Che server. " +
-                      "This error can usually be fixed with additional configuration settings in /conf/che.properties. " +
-                      "The Che server will stop this workspace after a short timeout. " +
-                      "You can get help by posting your config, stacktrace and workspace /etc/hosts below as a GitHub issue.");
+            LOG.error(WSocketEventBusClient.WORKSPACE_AGENT_STOPPED_MESSAGE);
```
Comment (skabashnyuk) [inline wsagent/wsagent-local/src/main/java/org/eclipse/che/ApiEndpointAccessibilityChecker.java]: log. warn

**Review by skabashnyuk — APPROVED** [review]
ok if @vparfonov ok

**Review by vparfonov — APPROVED** [review]

**Pull request merged**